PowerShell Scanners/Get Bitlocker information/Bitlocker Information.ps1

```diff
@@ -0,0 +1,5 @@
+if (-Not(Get-Command Get-BitlockerVolume -ErrorAction SilentlyContinue)) {
+    throw "Unable to find Get-BitlockerVolume. Available on Windows 10 or Server 2016 with the Bitlocker feature installed"
+}
```

Comment:
Could you change the if block over to? `#Requires -Modules BitLocker`

Reply:
Sure